test/shopify-cli/app_types/node_test.rb

```diff
@@ -104,6 +104,8 @@ def setup
       end
 
       def test_server_command
+        ShopifyCli::Tasks::Tunnel.stubs(:call)
+        ShopifyCli::Tasks::UpdateWhitelistURL.stubs(:call)
```

Comment:
this could be `expects(:call)` because we want to ensure that the server command updates the whitelist.